trace/sql/sql.go

```diff
@@ -373,3 +386,34 @@ func (tx *Tx) Stmt(ctx context.Context, stmt *Stmt) *Stmt {
 	defer trace.SpanSuccess(sp)
 	return &Stmt{stmt: tx.tx.StmtContext(ctx, stmt.stmt)}
 }
+
+// ParseDSN unpacks the connections string to the relevant struct
+func ParseDSN(dsn string) DSNInfo {
```

Comment:
There is probably no need for an exported function with a comment. Let's unexport and remove the comment.
BTW Comment should always end with a punctuation like sentences.

Reply:
I thought we'd export it, in case we find other uses for the rest of the fields (eg. driver or protocol). 
I'm unexporting the function, but should we also tweak it so we don't fetch additional data?